packages/shared/CHANGELOG.md

```diff
@@ -22,26 +22,68 @@
 * authorization token (JWT) bb-10 ([#40](https://github.com/BinaryStudioAcademy/bsa-2024-bebalance/issues/40)) ([16e3c35](https://github.com/BinaryStudioAcademy/bsa-2024-bebalance/commit/16e3c353ff700ab27a7b6af4fa7b3c17059cc916))
 
 ## [1.2.0](https://github.com/BinaryStudioAcademy/bsa-2024-bebalance/compare/shared-v1.1.0...shared-v1.2.0) (2024-08-21)
+>>>>>>> f9749449ad8a29e69c8f5599fce32330826a92fd
```

Comment:
I think it is left over from the resolution of the conflict? And I think we should not modify this file.